package.json

```diff
@@ -123,7 +123,7 @@
     "@elastic/elasticsearch": "^7.3.0",
     "archiver": "^3.0.0",
     "ava": "^3.13.0",
-    "aws-sdk": "^2.290.0",
+    "aws-sdk": "^2.1001.0",
```

Comment:
Match the requirement in alpha ^4.1.0. The version deployed to AWS lambdas is currently 2.1374.0 so this version is safe.